cmd/create/cluster/cmd.go

```diff
@@ -2362,12 +2383,20 @@ func handleByoOidcOptions(r *rosa.Runtime, cmd *cobra.Command, isSTS bool) (bool
 	return isByoOidcSet, oidcEndpointUrl, oidcPrivateKeySecretArn
 }
 
-func minReplicaValidator(multiAZ bool) interactive.Validator {
+func minReplicaValidator(multiAZ bool, isHostedCP bool, privateSubnetsCount int) interactive.Validator {
 	return func(val interface{}) error {
 		minReplicas, err := strconv.Atoi(fmt.Sprintf("%v", val))
 		if err != nil {
 			return err
 		}
+		if isHostedCP {
+			if minReplicas%privateSubnetsCount != 0 {
```

Comment:
Do we need to check that `minReplica` is a positive number?

Reply:
Yeah, I thought it was validated before, but not, added a condition there.
Thanks